Makefile

```
@@ -161,6 +161,9 @@ golangci-lint: ## Run golangci-lint on codebase
 
 	@echo Running golangci-lint
 	golangci-lint run
```

Nit: Can we replace this with
```suggestion
	golangci-lint run ./...
```

This make it a bit more clear to me, what it does.